docs/tutorials/edc-cut-bell-pair-benchmarking.ipynb

```diff
@@ -0,0 +1,715 @@
+    "* Measure Bell pair stabilizers: We then split into two versions of the circuit. In the first version, we  measure the $ZZ$ stabilizer on qubits 0 and 3. In the second version, we measure the $XX$ stabilizer on these qubits.\n",
+    "\n",
+    "For each 4-qubit initial layout, the above function returns two circuits (one for $ZZ$, one for $XX$ stabilizer measurement). At the end of this step, we have a list of circuits covering every 4-qubit chain on the device. These circuits include mid-circuit measurements and conditional (if/else) operations, which are the key instructions of the dynamic circuit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 63,
+   "id": "bd04755f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Image src=\"/docs/images/tutorials/edc-cut-bell-pair-benchmarking/extracted-outputs/bd04755f-0.avif\" alt=\"Output of the previous code cell\" />"
+      ]
+     },
+     "execution_count": 63,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuits = create_bell_stab(initial_layouts)\n",
+    "circuits[-1].draw(\"mpl\", fold=-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97eb3019",
+   "metadata": {},
+   "source": [
+    "## Step 2: Optimize problem for quantum hardware execution\n",
+    "\n",
+    "\n",
+    "Before executing our circuits on real hardware, we need to transpile them to match the device’s physical constraints. Transpilation will map the abstract circuit onto the physical qubits and gate set of the chosen device. Since we have already chosen specific physical qubits for each chain (by providing an `initial_layout` to the circuit generator), we use transpiler `optimization_level=0` with that fixed layout. This tells Qiskit not to reassign qubits or perform any heavy optimizations that could alter the circuit structure. We want to keep the sequence of operations (especially the conditional gates) exactly as specified.\n",
+    "\n",
+    "One important preparation is to ensure the transpiler knows about the dynamic circuit instructions. We explicitly add the `IfElseOp` (if/else construct) to the backend’s instruction set (`backend.target`) before transpiling. This lets the transpiler and scheduler know that our hardware can handle the conditional operations natively."
```

Comment:
Do we still have HW that doesn't support DC?  I thought they all supported them now.

Reply:
@jyu00 ?

Reply:
None of our hardware actually supports DC by default now 😅, because we decided to extend the early access period of the new version. So the only way to run DC jobs is through early access. 

This sentence `One important preparation is to ensure the transpiler knows about...` is not necessary. If a backend supports DC by default, it'd already have `IfElseOp` in the supported instructions. The only time we need to add that is during early access,